sqlitecluster/SQLiteNode.h

```diff
@@ -76,6 +76,10 @@ class SQLiteNode : public STCPManager {
         NUM_CONSISTENCY_LEVELS
     };
 
+    // This is a globally accessible pointer to some node instance. The intention here is to let signal handling code attempt to kill outstanding
+    // peer connections on this node before shutting down.
+    static SQLiteNode* KILLABLE_SQLITE_NODE;
```

Comment:
Is it important that this be named `KILLABLE_`? That hints at the intended use but nothing about this variable makes it killable per se, it's just a global reference. So would it make sense to use something more like `THIS_SQLITE_NODE`?